CoffeeChat/Controller/OnboardingPageViewController.swift

```diff
@@ -17,15 +17,33 @@ protocol OnboardingPageDelegate: class {
 class OnboardingPageViewController: UIPageViewController {
 
     // MARK: - Private View Vars
+    private let backgroundImage = UIImageView()
     private var demographicsViewController: DemographicsViewController!
     private var groupsViewController: GroupsViewController!
     private var interestsViewController: OnboardingInterestsViewController!
     private var onboardingPages = [UIViewController]()
 
+    // MARK: - Private Data Vars
+    private var backgroundXPosition: CGFloat = UIScreen.main.bounds.width
+    private let screenWidth = UIScreen.main.bounds.width
+    private let screenHeight = UIScreen.main.bounds.height
+
     override func viewDidLoad() {
         super.viewDidLoad()
         navigationController?.navigationBar.isHidden = true
 
+        for subview in view.subviews {
+            if let scrollView = subview as? UIScrollView {
+                scrollView.delegate = self
+            }
+        }
+
+        backgroundImage.image = UIImage(named: "onboardingBackground")
+        backgroundImage.contentMode =  .scaleAspectFill
+        view.insertSubview(backgroundImage, at: 0)
+        print(backgroundXPosition)
+        backgroundImage.frame = CGRect(x: backgroundXPosition, y: 0, width: screenWidth, height: screenHeight)
```

Comment:
we can't use constraints?

Reply:
hmm here, im setting the start of the background at a specific value and then I keep updating it in scrollViewDidScroll. is there a better way to do this?